views/about.erb

```diff
@@ -0,0 +1,40 @@
+<html>
+<body>
+
+<head>
+  <title>About Me</title>
+</head>
+
+<header>
+<h1><img src="http://www.cliparthut.com/clip-arts/1778/cat-head-clip-art-1778708.jpg" height=25 width = 25>
```

Comment:
With Ruby, we've learned to add space before/after the `=`, but with HTML we avoid adding that "extra" space